toil-appliance/test/test.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+mkdir ./tempDir
+
+LOCAL_PATH=$(cd ./tempDir; pwd)
+LEADER_PATH=/test/data
+UUID=$(python -c 'import uuid; print uuid.uuid4()')
+LEADER_IP=172.17.0.2
+LEADER_PORT=5050
+LEADER=toil-leader-$UUID
+WORKER=toil-worker-$UUID
+
+echo Starting containers
+docker run -d -v $LOCAL_PATH:$LEADER_PATH --name $LEADER quay.io/ucsc_cgl/toil-leader --ip=$LEADER_IP \
+--registry=in_memory
```

Comment:
Nit: If you're splitting lines, might as well split out more lines and indent the split out lines.